### PR TITLE
Fixes encoding characters that come in contact fields

### PR DESCRIPTION
**Pull request by mqueme**
[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | y
| New feature? | 
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
When creating Contacts through the UI, the API and through integrations and imports. Values that may have certain characters like & get encoded, resulting in various errors.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. create a user through the API or the UI, containing an & sign
2. check in your database and see encoded value

#### Steps to test this PR:
1. Apply this PR and create a new contact with ampersand or edit existing one
2. Look in your database and values should not be encoded

### CODE REVIEW

File: app/bundles/LeadBundle/Form/Type/EntityFieldsBuildFormTrait.php [hunk 1]
```diff
@@ -221,14 +221,15 @@ private function getFormFields(FormBuilderInterface $builder, array $options, $o
                             break;
                         case 'email':
                             // Enforce a valid email
-                            $constraints[] = new Email(
+                            $attr['data-encoding'] = 'email';
```
Comment (cklingsporn) [inline app/bundles/LeadBundle/Form/Type/EntityFieldsBuildFormTrait.php]: Does this get over riden by ln232 without doing anything?

Comment (mqueme) [inline app/bundles/LeadBundle/Form/Type/EntityFieldsBuildFormTrait.php]: There is a break in 230,so no

Comment (mqueme) [inline app/bundles/LeadBundle/Form/Type/EntityFieldsBuildFormTrait.php]: No

**Comment by virlatinus** [comment]
The ampersands are not encoded anymore. +1

**Pull request merged**